.github/workflows/test-and-lint.yml

```diff
@@ -13,7 +13,7 @@ permissions:
 
 jobs:
   backend_build:
-    name: Backend Lint and Tests
```

Comment:
If we're not handling linting in this workflow anymore then we should change the file name

Reply:
We're still linting just no in this particular subtask.